agent/acs/handler/payload_handler_test.go

```diff
@@ -45,9 +46,10 @@ func TestHandlePayloadMessageWithNoMessageId(t *testing.T) {
 	ecsClient := mock_api.NewMockECSClient(ctrl)
 	stateManager := statemanager.NewNoopStateManager()
 	credentialsManager := credentials.NewManager()
+	taskHandler := eventhandler.NewTaskHandler()
 
 	ctx := context.Background()
-	buffer := newPayloadRequestHandler(ctx, taskEngine, ecsClient, clusterName, containerInstanceArn, nil, stateManager, refreshCredentialsHandler{}, credentialsManager)
+	buffer := newPayloadRequestHandler(ctx, taskEngine, ecsClient, clusterName, containerInstanceArn, nil, stateManager, refreshCredentialsHandler{}, credentialsManager, taskHandler)
```

Comment:
This too, might benefit from splitting into multiple lines. There are multiple such occurrences in the diff and you might want to change all of those.